chromaui/test-archiver
# Truncate and map filenames that are too long in test archiver

Issue: CAP-1203

## What Changed

Sometimes, filenames are too long to be written to the file system. Typically, this happens if a file name is longer than 256 bytes. When this happens, we want to truncate the file name and map references to this file to the new, shortened filename.

## How to test

There are tests for this in `write-archive/index.test.ts`, but you can follow these steps to reproduce:

1. Link this branch so that you can use `test-archiver` locally.
2. Clone https://github.com/tevanoff/chromatic-e2e-issues
3. Run yarn playwright test toolong.test.ts
4. Verify that no error occurs.

## Change Type
